src/main/flight/pid.c

```diff
@@ -704,6 +709,9 @@ void pidInitConfig(const pidProfile_t *pidProfile)
 #endif
 }
 
+uint8_t pidAdjustmentChannel;
```

Comment:
These should be `static`. And because you are accessing them from `FAST_CODE`, they also should be `FAST_RAM_ZERO_INIT`.